Concretização do componente UI de um subssistema de avaliação segundo as propriedades especificadas

michelam-pseudo opened the pull request.

Commit:
Concretização do mockup de um sistema de avaliação segundo as propriedades especificadas

mateacademy-ai-mentor reviewed (commented):
Sorry, the pull request review was unsuccessful for the following reasons:

> [!WARNING]
> **Too many changes in the pull request.** Please make sure that you have added only the necessary changes to the pull request.
> <sub>*[CODE: 6]*</sub>


If the issue is within your control, please address it.
The next review attempt <ins>will automatically launch</ins> in 1 hour.
If you believe this is an error or need assistance, please contact the support team.

Commit:
Refinamento corretivo para satisfação do requisito de blocos BEM não monolíticos; configuração do arquivo readme no que toca aos parâmetros de deploy

mateacademy-ai-mentor reviewed (commented):
Sorry, the pull request review was unsuccessful for the following reasons:

> [!WARNING]
> **Too many changes in the pull request.** Please make sure that you have added only the necessary changes to the pull request.
> <sub>*[CODE: 6]*</sub>


If the issue is within your control, please address it.
The next review attempt <ins>will automatically launch</ins> in 1 hour.
If you believe this is an error or need assistance, please contact the support team.

mateacademy-ai-mentor reviewed (commented):
Sorry, the pull request review was unsuccessful for the following reasons:

> [!WARNING]
> **Too many changes in the pull request.** Please make sure that you have added only the necessary changes to the pull request.
> <sub>*[CODE: 6]*</sub>


If the issue is within your control, please address it.
The next review attempt <ins>will automatically launch</ins> in 1 hour.
If you believe this is an error or need assistance, please contact the support team.

mateacademy-ai-mentor reviewed (commented):
Sorry, the pull request review was unsuccessful for the following reasons:

> [!WARNING]
> **Too many changes in the pull request.** Please make sure that you have added only the necessary changes to the pull request.
> <sub>*[CODE: 6]*</sub>


If the issue is within your control, please address it.
The next review attempt <ins>will automatically launch</ins> in 1 hour.
If you believe this is an error or need assistance, please contact the support team.

pedro-ruas reviewed (commented):
Remove .js files, add the html code inside html.
Remember to run `npm run deploy` and add your DEMO link and TEST link to the PR comment

pedro-ruas reviewed (changes requested):
Remember to run `npm run deploy` and add the DEMO link and TEST link to the PR

Commit:
Deleção do diretório após especificação de correção necessária.

Commit:
Involução de estruturação por HTML.

Atualização da estruturação dos blocos BEM por linguagem estrutural de maneira direta.

michelam-pseudo commented on src/blocks/stars--0.js:
Okay. I've returned to HTML structuring.

michelam-pseudo commented:
> Remember to run `npm run deploy` and add the DEMO link and TEST link to the PR
